service/service.go

```diff
@@ -258,6 +258,7 @@ func (s *Service) UpdateUser(ctx context.Context, req *pb.UserRequest) (*pb.User
 	logger.Info("Updated user:", updatedUser.GetUuid(),
 		updatedUser.GetFirstName(), updatedUser.GetLastName())
 
+	updatedUser.Password = ""
```

Comment:
Is there where password gets reset to `""`?

Reply:
yes